compiler/include/resolution.h

```diff
@@ -146,7 +146,8 @@ void wrapAndCleanUpActuals(ResolutionCandidate* best, CallInfo& info,
 typedef enum {
   FIND_EITHER = 0,
   FIND_REF,
-  FIND_NOT_REF
+  FIND_CONST_REF,
+  FIND_NOT_REF_OR_CONST_REF,
```

Comment:
Ah, OK, makes sense. I don't think I have a better name. Maybe just a comment with that logic, but that's optional.